xml/System.Net.Security/SslStream.xml

```diff
@@ -3453,9 +3453,17 @@ Authentication has not occurred.</exception>
         <ReturnType>System.Net.Security.TlsCipherSuite</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the cipher suite resolved to use by this <see cref="T:System.Net.Security.SslStream"/>.</summary>
```

Comment:
This is a cipher suite which was already negotiated and is already being used. Comment could be something like:

`Cipher suite picked by the server from the client supplied list of allowed cipher suites`

Reply:
with `Gets` prefix per @bartonjs comment